mllib/src/main/scala/org/apache/spark/ml/r/LDAWrapper.scala

```diff
@@ -123,6 +126,10 @@ private[r] object LDAWrapper extends MLReadable[LDAWrapper] {
       .setMaxIter(maxIter)
       .setSubsamplingRate(subsamplingRate)
 
+    if (optimizer == "em") {
```

Comment:
If it is "online", it is not necessary to set the optimizer. But setting it anyway will make the code clean. I will do it.